src/containers/guest-home-page/signup-form/SignupForm.jsx

```diff
@@ -37,6 +38,16 @@ const SignupForm = ({
     setIsAgreementChecked((prev) => !prev)
   }
 
+  const handleOnInput = () => {
+    setConfirmPasswordClicked(true)
+  }
+
+  const confirmPasswordDoesNotMatchError =
```

Comment:
Take a look at the files `validation/login.js` and `hooks/use-form.jsx.` If I'm not mistaken, we already have all the necessary functions for validation, we just need to add them to the fields. Maybe it's better to use them? You can check how it was implemented in `LoginForm.jsx` and `LoginDialog.jsx`.

Reply:
The purpose of the bug is as follows:

Expected result:
The red error text 'Passwords do not match' should appear under the 'Confirm Password' field when it is filled with invalid data.

I have implemented the logic so that the red error text 'Passwords do not match' appears when both field values do not match, as can be seen in the picture.

I believe bug #77 is not related to the validation of the confirmPassword field on form submission. However, as you noticed, there is an error with the validation of the confirmPassword field.

I am going to create a new issue for the confirmPassword validation bug and create another branch to fix this error.

Reply:
My point was just that we already have all the functionality, including displaying error messages, we just need to add it to the fields. But, as usual, we can achieve the same result with different approaches. So it's great that everything is working as expected now. Good job!

Reply:
Thanks for your comment. However, I do not see any existing functionality that is related to checking the password field and confirPassword field for the same values and showing the error message. I created the logic for handling input and checking for password match errors that is specific to this form and unlikely to be reused elsewhere. Keeping the logic in the component can maintain a clearer separation of concerns.

Reply:
@iva-stasia says about already existed custom hook `useForm`, this hook can be used for validation all users inputs, including passwords. I think we can use this hook later for this component for valiadation all fields. 
It's interesting, that React has its own [use-form hook](https://react-hook-form.com/), but our project already has its own implementation.